HPXMLtoOpenStudio/resources/materials.rb

```diff
@@ -79,6 +79,12 @@ def self.AirFilmOutside
     return self.AirFilm(rvalue)
   end
 
+  def self.AirFilmMFSpaces
+    combined_film_u = 8.3
```

Comment:
We should reference this [here](https://github.com/NREL/OpenStudio-HPXML/blob/interior-surface-test/HPXMLtoOpenStudio/measure.rb#L3413) so that we don't have 8.3 hard-coded in two different places.